lib/src/components/board/board.dart

```diff
@@ -16,6 +18,13 @@ class BoardStatus {
     pbBoardStatus.digitalInterrupts.forEach((key, value) => boardStatus.digitalInterrupts[key] = (value.value.toInt()));
     return boardStatus;
   }
+
+  common.BoardStatus toPbBoardStatus() {
```

Comment:
I think this would be better as a ` get` attribute called `proto`

```suggestion
  common.BoardStatus get proto {
```

Reply:
i like that, good call.